addons/gamepad/userscript.js

```diff
@@ -227,11 +224,11 @@ export default async function ({ addon, global, console, msg }) {
   gamepad.addEventListener("mousemove", handleGamepadMouseMove);
 
   while (true) {
-    const stageHeaderWrapper = await addon.tab.waitForElement('[class*="stage-header_stage-menu-wrapper"]', {
+    const stageHeaderSizeControls = await addon.tab.waitForElement('[class*="stage-header_stage-size-row"]', {
```

Comment:
The changes this PR makes to the gamepad addon makes the settings button never appear in fullscreen mode. That's a bug.

Reply:
I will fix this in #2658. Would recommend reverting all the gamepad changes here because there's going to be conflicts otherwise.

Reply:
Done